src/js/components/Home.js

```diff
@@ -1,27 +1,21 @@
-import { select, templates } from '../settings.js';
+import { templates } from '../settings.js';
+import CarouselWidget from '../components/CarouselWidget.js';
 
 class Home {
-  constructor(element){
+  constructor(element) {
     const thisHome = this;
     thisHome.render(element);
-    thisHome.initWidgets();
+    CarouselWidget.initWidgets();
   }
   render(element) {
     const thisHome = this;
     const generatedHTML = templates.homeWidget();
-    console.log (generatedHTML);
+    console.log(generatedHTML);
     thisHome.dom = {};
-    thisHome.dom.wrapper= element;
+    thisHome.dom.wrapper = element;
     thisHome.dom.wrapper.innerHTML = generatedHTML;
   }
-  initWidgets() {
-    const element = document.querySelector(select.widgets.carousel);
-    new Flickity (element, {
-      autoPlay: 2500,
-      prevNextButtons: false,
-      imagesLoaded: true,
-    });
-  }
+  initWidgets()
```

Comment:
To wywołanie `initWidgets` nie odnosi się do niczego. Poza tym jest wywołane poza jakąkolwiek metodą. Usuń to wywołanie.